frameworks/AutoGluon/setup.sh

```diff
@@ -29,11 +30,15 @@ else
     rm -Rf ${TARGET_DIR}
     git clone --depth 1 --single-branch --branch ${VERSION} --recurse-submodules ${REPO} ${TARGET_DIR}
     cd ${TARGET_DIR}
-    PIP install -e core/
+    # Note: Normally we would just call `./full_install.sh` but because `pip` and `PIP` are not the same,
+    # the script does not work here. Therefore, we have to explicitly install each submodule below.
+    # This has the downside that source installs of old versions of the package might not follow the below steps.
+    # It is recommended to instead use pip install of older AG versions to ensure it works correctly.
+    # https://github.com/awslabs/autogluon/blob/master/full_install.sh
```

Comment:
hack alert! what about sth like:
```bash
env PATH=`dirname $py_exec`:$PATH  bash -c ./full_install.sh
```
haven't tried, but shouldn't it work?

Reply:
Thanks! This makes things a lot nicer and allows us to support AutoGluon v0.1.0+ all at once. Refer to the updated PR.